frontend/shared-components/info-tooltip/info-tooltip.module.css

```diff
@@ -0,0 +1,59 @@
+.tooltipContainer {
```

Comment:
is this the same info-tooltip from pages?

Reply:
I think GitHub's PR UI for some reason might've been showing you an outdated structure (I had originally had `shared-components/` inside of `pages/` but moved it a level higher before pushing for PR)

I do believe there is only one tooltip so far!